controller/seafile-controller.c

```diff
@@ -320,7 +320,7 @@ setup_env ()
     g_setenv ("SEAFILE_CENTRAL_CONF_DIR", ctl->central_config_dir, TRUE);
 
     char *seahub_dir = g_build_filename (installpath, "seahub", NULL);
-    char *seafdav_conf = g_build_filename (topdir, "conf", "seafdav.conf", NULL);
+    char *seafdav_conf = g_build_filename (ctl->central_config_dir, "seafdav.conf", NULL);
```

Comment:
The change doesn't make any functional difference, but it LGTM to use the `central_config_dir` variable here.